eng/pipelines/templates/steps/common.yml

```diff
@@ -14,5 +14,5 @@ steps:
     displayName: "Use NodeJS $(NodeVersion)"
 
   - script: |
-      npm --version
-    displayName: NPM version
+      npm install -g npm@6.9.0
+    displayName: "Install npm 6.9.0"
```

Comment:
Please revert the changes to this file to get the PR CI working in Mac.
The fix to the master branch was #6631 which I cherry-picked to the `feature/service-bus-atom` branch. This change here is reverting that fix.

Reply:
It needed to be manually updated during merge, might have got missed then - fixed in latest commit